netci.groovy

```diff
@@ -3,12 +3,18 @@ import jobs.generation.Utilities
 def project = GithubProject
 def branch = GithubBranchName
 def isPR = true
-def platformList = ['Ubuntu16.04:Debian', 'Windows_2016:NanoServer']
+def platformList = ['Ubuntu16.04:Debian', 'Windows_2016:NanoServer', 'Windows_2016:NanoServer-1709']
 
 platformList.each { platform ->
     def(hostOS, containerOS) = platform.tokenize(':')
     def machineLabel = (hostOS == 'Windows_2016') ? 'latest-docker' : 'latest-or-auto-docker'
-    def versionList = (hostOS == 'Windows_2016') ? ['1.', '2.0', '2.1'] : ['1.', '2.']
+
+    if (containerOS == 'NanoServer-1709') {
+        versionList = ['2.0', '2.1']
+    }
+    else {
```

Comment:
I think `if else if else` would be cleaner here.

Reply:
Ok. Updated as -

```groovy
    if (containerOS == 'NanoServer-1709') {
        versionList = ['2.0', '2.1']
    }
    else if (containerOS == 'NanoServer') {
        versionList = ['1.', '2.0', '2.1']
    }
    else {
        versionList = ['1.', '2.']
    }
```